### DISCUSSION TITLE
How to watch namespaces with specific pattern

### DISCUSSION THREAD

**Discussion by lnthai2002** [arkmq-org/arkmq-org-broker-operator#548] [Q&A] [answered]
Hello,
How do i specify WATCH_NAMESPACE to watch all namespaces with specific pattern for example customer-ns-{9 digits}. 

Thanks
Thai Le

**Comment by brusdev** [comment, marked as answer]
The WATCH_NAMESPACE env is a list of namespaces comma separated it doesn't support patterns.
